packages/cli/src/PublicApi/v1/handlers/workflows/workflows.handler.ts

```diff
@@ -179,7 +179,7 @@ export = {
 			await replaceInvalidCredentials(updateData);
 			addNodeIds(updateData);
 
-			const workflowRunner = Container.get(ActiveWorkflowRunner);
+			const workflowRunner = Container.get(ActiveWorkflowManager);
```

Comment:
```suggestion
			const workflowManager = Container.get(ActiveWorkflowManager);
```

Should we rename this as well? Or would that make the PR too big?